MediaGalleryUi/etc/di.xml

```diff
@@ -15,6 +15,20 @@
             </argument>
         </arguments>
     </type>
+     <type name="Magento\MediaGalleryUi\Model\FilesIndexer">
+         <arguments>
+	      <argument name="excludedDirectories" xsi:type="array">
```

Comment:
I think it might make sense to introduce a single class providing the excluded directories, configure it via DI and use this single class in both FilesIndexer and FolderTree

Reply:
ok

Reply:
@sivaschenko done